### PR TITLE
feat: move landing to root, reorder datasets/collections

### PR DESCRIPTION
**Pull request by seve**
#2992 #2991

### Reviewers
**Functional:**  @MillenniumFalconMechanic

### CODE REVIEW

**Comment by seve** [comment]
@MillenniumFalconMechanic Do you know if any of the links surrounding the breadcrumbs on the explorer will have to be changed in order to support any of these changes? I know a link is passed via [the config here](https://github.com/chanzuckerberg/single-cell-explorer/blob/1b10f5ed62715ee3ecce1a352765c3a069f4db5d/client/src/globals.ts#L31-L47), but I'm not sure what is actually expected/sent

File: frontend/src/components/Layout/index.tsx [hunk 1]
```diff
@@ -9,8 +10,9 @@ import { Wrapper } from "./style";
 const Layout: FC = ({ children }) => {
   const { pathname } = useRouter();
 
-  // CHANGE TO "/" FOR PROD
-  if (pathname === "/landing-page") {
+  console.log(pathname);
```
Comment (MillenniumFalconMechanic) [inline frontend/src/components/Layout/index.tsx:13]: Do we want to keep the `console.log` here?

Comment (seve) [reply frontend/src/components/Layout/index.tsx:13]: TY!

Comment (seve) [reply frontend/src/components/Layout/index.tsx:13]:
```suggestion
```

**Review by MillenniumFalconMechanic — APPROVED** [review]
LGTM Seve! One q: should we update the [tombstone redirect](https://github.com/chanzuckerberg/single-cell-data-portal/blob/seve/edit-sitemap/frontend/src/views/Collection/index.tsx#L91) to `/collections`? Or will we handle this update in [Refine and update tombstoned collection toast functionality](https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell-data-portal/2926)?

**Comment by MillenniumFalconMechanic** [comment]
> @MillenniumFalconMechanic Do you know if any of the links surrounding the breadcrumbs on the explorer will have to be changed in order to support any of these changes? I know a link is passed via [the config here](https://github.com/chanzuckerberg/single-cell-explorer/blob/1b10f5ed62715ee3ecce1a352765c3a069f4db5d/client/src/globals.ts#L31-L47), but I'm not sure what is actually expected/sent

The breadcrumbs in Explorer should link to:

Collection Index Page > Collection Detail Page > Dataset Dropdown

The link to the collection index page is pulled from `links.collections-home-page` returned in in the `config` response and will require a backend change to update this to `origin/collections`. No frontend is required here.

Screen Shot 2022-07-29 at 10 34 46 AM

The link to the collection detail page is pulled from `collection_url` returned in the `/dataset-metadata` response and doesn't require updating.

**Comment by seve** [comment]
> LGTM Seve! One q: should we update the [tombstone redirect](https://github.com/chanzuckerberg/single-cell-data-portal/blob/seve/edit-sitemap/frontend/src/views/Collection/index.tsx#L91) to `/collections`? Or will we handle this update in [Refine and update tombstoned collection toast functionality](https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell-data-portal/2926)?

Let's address this in a seperate PR @MillenniumFalconMechanic

**Pull request merged**